sklearn/multioutput.py

```diff
@@ -196,6 +196,9 @@ def predict(self, X):
 
         return np.asarray(y).T
 
+    def _more_tags(self):
+        return {'multioutput_only': True}
```

Comment:
These do not seem to inherit from MultiOutputMixin, but maybe I'm missing something (and we should probably have a check for the integrity of tags such as multioutput_only => multioutput and multioutput & classifier => multilabel)